source/agora/common/Data.d

```diff
@@ -26,4 +26,12 @@ public alias Hash = BitBlob!256;
 public alias Address = string;
 
 /// The type of a signature
-public alias Signature = BitBlob!256;
+public alias Signature = BitBlob!512;
+
+unittest
```

Comment:
Ah.. but this static assert will not be compiled-in if `-unittest` is not passed. Would we consider this a problem?